lib/constants/diff-modes.ts

```diff
@@ -3,38 +3,38 @@ import {ValueOf} from 'type-fest';
 export const DiffModes = {
     THREE_UP: {
         id: '3-up',
-        title: '3-up',
-        description: 'images in column'
+        title: 'List',
+        description: 'List. Show images one after another in vertical layout.'
     },
     THREE_UP_SCALED: {
         id: '3-up-scaled',
-        title: '3-up scaled',
-        description: 'scaled images in row'
+        title: 'SbS',
+        description: 'Side by Side. Show images in one row.'
     },
     THREE_UP_SCALED_TO_FIT: {
         id: '3-up-scaled-to-fit',
-        title: '3-up scaled to fit',
-        description: 'scaled to browser height images in row'
+        title: 'SbS (fit screen)',
+        description: 'Side by Side. Show images in one row and scale them down if needed to fit the screen.'
     },
     ONLY_DIFF: {
         id: 'only-diff',
-        title: 'Only diff',
-        description: 'click on image to see area with diff'
+        title: 'Only Diff',
+        description: 'Only Diff. Show only diff image, click to highlight diff areas.'
     },
     SWITCH: {
         id: 'switch',
         title: 'Switch',
-        description: 'click on image to switch'
+        description: 'Switch. Click to switch between expected and actual images.'
```

Comment:
I think we don't need to duplicate the name here

Reply:
I think we do, because for instance we have SbS label and Side by Side as an explanation. Keeping it for now.